app/models/user/MultiUser.scala

```diff
@@ -0,0 +1,94 @@
+package models.user
+
+import com.mohiva.play.silhouette.api.util.PasswordInfo
+import com.scalableminds.util.accesscontext.DBAccessContext
+import com.scalableminds.util.tools.Fox
+import javax.inject.Inject
+import slick.jdbc.PostgresProfile.api._
+import com.scalableminds.webknossos.schema.Tables._
+import slick.lifted.Rep
+import utils.{ObjectId, SQLClient, SQLDAO}
+
+import scala.concurrent.ExecutionContext
+
+case class MultiUser(
+    _id: ObjectId,
```

Comment:
Is there a reason, you did not put the first- and lastname into the multi user?

Reply:
Yes, though I’m not 100% convinced myself. There may be situations where a user wants to have specific names depending on the organization (e.g. not share their full name with one). It also made for the smaller change right now. We may want to change this later, though. Or if you think this doesn’t make sense, I’ll also change it here.